cql3/query_processor.hh

```diff
@@ -446,7 +446,7 @@ public:
 
     void reset_cache();
 
-    bool topology_global_queue_empty();
+    future<> alter_tablets_keyspace();
```

Comment:
I added this method here, because `remote` is only declared in this scope, and defined in the corresponding .cc file, so this method just calls a similar method from storage_service without exposing storage_services's details.